components/Authentication/login.jsx

```diff
@@ -147,14 +154,27 @@ const Login = ({ navigation }) => {
         </Button>
       </View>
       <ViewFiller />
-      <LoginButton
-        uppercase={false}
-        mode="contained"
-        dark
-        onPress={loginPress}
-      >
-        <LoginText>Login</LoginText>
-      </LoginButton>
+      {!disableButton
+        ? (
+          <LoginButton
+            uppercase={false}
+            mode="contained"
+            dark
+            onPress={loginPress}
+          >
+            <LoginText>Login</LoginText>
+          </LoginButton>
+        )
+        : (
+          <DisableLoginButton
```

Comment:
Would u want this button to have a disabled={disableButton} like you do the sign up one?

Reply:
yeah good catch